### PR TITLE
Stable diffusion models

### PR DESCRIPTION
**Pull request by rjaincc**
text-to-image Models:
1. runwayml/stable-diffusion-v1-5 - https://ml.azure.com/experiments/id/310732e9-14db-4e9e-a1d7-d19233140c42/runs/b2d00457-ecda-459c-8790-4372e068fd70?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/grajguru-rg/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

2. stabilityai/stable-diffusion-2-1 - https://ml.azure.com/experiments/id/310732e9-14db-4e9e-a1d7-d19233140c42/runs/ad795e0c-a2e8-4171-b253-d90e5fd2215d?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/grajguru-rg/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

3. CompVis/stable-diffusion-v1-4 - https://ml.azure.com/experiments/id/310732e9-14db-4e9e-a1d7-d19233140c42/runs/1f83b909-7b29-45cc-b9ea-ba77c590e0e1?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/grajguru-rg/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

text-to-image inpainting models:
1. stabilityai/stable-diffusion-2-inpainting - https://ml.azure.com/experiments/id/310732e9-14db-4e9e-a1d7-d19233140c42/runs/86f44b11-c6e0-4098-92bf-117b2f19f04d?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/grajguru-rg/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

2. runwayml/stable-diffusion-inpainting - https://ml.azure.com/experiments/id/310732e9-14db-4e9e-a1d7-d19233140c42/runs/af82d3c0-d5d7-463d-8dfd-7c8d4b41e62f?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/grajguru-rg/workspaces/grajguru-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

Please note that runs are from Madhu.

### CODE REVIEW

File: assets/models/system/stabilityai-stable-diffusion-2-1/description.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+
+```json
+{
+   "input_data": {
+        "columns": ["prompt"],
+        "data": ["a photograph of an astronaut riding a horse"],
+        "data": ["a photograph of an astronaut riding a horse"],
```
Comment (vivek-dani) [inline assets/models/system/stabilityai-stable-diffusion-2-1/description.md:28]: data field appears twice

File: assets/models/system/runwayml-stable-diffusion-v1-5/description.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "prompt": "lion holding hunted deer in grass fields",
+        "generated_image": "image2",
+        "nsfw_content_detected": True
+    }
+]
+```
+
+> Note:
+>
+> - "image1" and "image2" string are base64 format.
```
Comment (vivek-dani) [inline assets/models/system/runwayml-stable-diffusion-v1-5/description.md:52]: strings

File: assets/models/system/stabilityai-stable-diffusion-2-1/description.md (hunk 1 above)
Comment (rjaincc) [reply assets/models/system/stabilityai-stable-diffusion-2-1/description.md:28]: Fixed

File: assets/models/system/stabilityai-stable-diffusion-2-1/description.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+The Stable Diffusion v2-1 model is a fine-tuned version of the Stable Diffusion v2 model, with additional training steps on the same dataset. It's designed for generating and modifying images based on text prompts, utilizing a Latent Diffusion Model with a fixed, pretrained text encoder (OpenCLIP-ViT/H). The model was trained on the LAION-5B dataset and its subsets, with further filtering using LAION's NSFW detector with a conservative "p_unsafe" score of 0.1. The model has various applications in research, art, education, and creative tools. However, there are strict guidelines for the model's use to prevent misuse and malicious activities. It should not be used to create harmful, offensive, or discriminatory content. Additionally, the model has limitations, such as difficulties with photorealism, rendering legible text, and generating complex compositions. The model was primarily trained on English descriptions, potentially leading to biases and limited effectiveness with non-English prompts. To enhance safety, a Safety Checker is recommended for use with this model.
```
[the remark this answers is not in the document]
Comment (rjaincc) [reply assets/models/system/stabilityai-stable-diffusion-2-1/description.md:1]: Sure, we can do that

[the remark this answers is not in the document]
Comment (rjaincc) [reply assets/models/system/stabilityai-stable-diffusion-2-1/description.md:1]: Done

**Review by vivek-dani — DISMISSED** [review]

**Review by haritha-bendapudi — APPROVED** [review]

**Review by haritha-bendapudi — DISMISSED** [review]

**Review by MadhuM02 — DISMISSED** [review]

**Review by vivek-dani — DISMISSED** [review]

**Review by haritha-bendapudi — DISMISSED** [review]

**Review by vivek-dani — DISMISSED** [review]

**Review by haritha-bendapudi — APPROVED** [review]

**Review by MadhuM02 — APPROVED** [review]

**Review by raduk — APPROVED** [review]

**Pull request merged**